packages/python3.xlmmacrodeobfuscator.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = "XLMMacroDeobfuscator"
+  $category = "Office"
+
+  $whlUrl = "https://github.com/DissectMalware/$toolName/releases/download/v0.2.7/$toolName-0.2.7-py3-none-any.whl"
```

Comment:
is there a reason we can't use the version of pypi: https://pypi.org/project/XLMMacroDeobfuscator/ (just `py -3.10 -m pip install XLMMacroDeobfuscator`)?

Reply:
@KJaeYoung do we need a shortcut in the Tools folder for XLMMacroDeobfuscator?

Reply:
If we don't need the shortcut and we can use the version of pypi, I think we should add this tool to python3.libraries.vm config instead of creating a new package. 🤔

Reply:
> @KJaeYoung do we need a shortcut in the Tools folder for XLMMacroDeobfuscator?

Nope, no need for a shortcut in the Tools folder. Just needs to be accessible via the command line.

Reply:
No specific reason, both techniques are the same.

Reply:
pypi handles the version updates for us without having to deal with hashes, so I think we should use the Pypi version as we are doing for other python libraries.

Reply:
Thanks for the answer @KJaeYoung.

Two options then:
1. Leave this as a separate package with the shortcut
2. Add this package to the [python3.libraries.vm modules file](https://github.com/mandiant/VM-Packages/blob/main/packages/libraries.python3.vm/tools/modules.xml) instead of creating a new package

option 1 gives us more flexibility and the shortcut, while option 2 simplifies the code and keeps it consistent with the current python libraries. I don't really have a strong opinion here. @mandiant/flare-vm @d35ha any preferences?

Reply:
@Ana06 
I prefer the second option, more consistent and minimal changes.

Reply:
@Ana06 
Added to the python libraries list.